SIT.Manager.Avalonia/Views/SettingsPage.axaml

```diff
@@ -22,74 +23,88 @@
 		<ScrollViewer Grid.Row="0">
 			<StackPanel Margin="10">
 				<!--Launcher settings-->
-				<TextBlock Text="Launcher Settings" Margin="5" VerticalAlignment="Center"/>
+				<TextBlock Text="{DynamicResource SettingsLauncherSettingsTitle}" Margin="5" VerticalAlignment="Center"/>
 				<Border CornerRadius="8" BorderBrush="Gray" BorderThickness="1" Padding="10">
 					<StackPanel>
-						<CheckBox Content="Close Launcher After Game Launch" 
-								  ToolTip.Tip="Closes the launcher automatically after opening the game."
+						<CheckBox Content="{DynamicResource SettingsCloseLauncherAfterGameLaunchTitle}" 
+								  ToolTip.Tip="{DynamicResource SettingsCloseLauncherAfterGameLaunchToolTip}"
 								  IsChecked="{Binding Config.CloseAfterLaunch, Mode=TwoWay}"/>
-						<CheckBox Content="Look For Updates" 
-								  ToolTip.Tip="Automatically look for updates when opening the launcher."
+						<CheckBox Content="{DynamicResource SettingsLookForUpdatesTitle}" 
+								  ToolTip.Tip="{DynamicResource SettingsLookForUpdatesTitleToolTip}"
 								  IsChecked="{Binding Config.LookForUpdates, Mode=TwoWay}"/>
+						<Grid>
+							<Grid.ColumnDefinitions>
+								<ColumnDefinition Width="auto"/>
+								<ColumnDefinition/>
+							</Grid.ColumnDefinitions>
```

Comment:
This can be condense into `<Grid ColumnDefinitions="auto, *">`

Reply:
Sugar syntax but okay 😄